README.md

```diff
@@ -716,6 +717,19 @@ __AWESOME__ apps counter: __253__ 🎉
 - [x] [GitHub](https://github.com/Pool-Of-Tears/Myne)
 - [ ] Official page
 
+## Education
+
+### Quiz-Platform
+
+> Quiz Platform is a Open Source knowledge testing platform for Android based on Jetpack Compose and Google solutions.
+
+<img alt="QuizPlatformIcon" height="64" src="https://raw.githubusercontent.com/Yugyd/quiz-platform/master/app/src/main/res/mipmap-hdpi/ic_launcher.webp">
+
+- [x] [Google Play]()
+- [ ] F-Droid
```

Comment:
Shame there is no F-Droid or IzzyOnDroid. However, the source code repo with releases with [Obtainium](https://github.com/ImranR98/Obtainium) is fine, I guess.

Reply:
Thank you, I have added a new release to Github - https://github.com/Yugyd/quiz-platform/releases/tag/v1.0.1

Reply:
Thanks, one last thing here - I see on Google Play "Contains ads". Does this .apk file have any ads? We have an "Ad-free" rule, so you know.